src/gpuarray_blas_opencl_clblas.c

```diff
@@ -56,6 +56,10 @@ static void teardown(gpucontext *ctx) {
     clblasTeardown();
 }
 
+static const char *error(gpucontext *ctx) {
+  return "(clblas) error in blas call, no details for now.";
```

Comment:
I think this is a bad sentence. Even if there is no error, we tell there is an error. What about:

(clblas) We didn't implement the detailed error message. Maybe there is an error, maybe there isn't.